kubernetes/config/kube_config.c

```diff
@@ -21,15 +24,34 @@ static int setBasePath(char **pBasePath, char *basePath)
     return -1;
 }
 
-static char *kubeconfig_mk_cert_key_tempfile(const char *b64data)
+static int is_cert_or_key_base64_encoded(const char *data)
```

Comment:
I think we can assume `c99` at this point, so `static bool is_cert_or_key_base64_encoded(...)`

Reply:
OK.